python/setup.py

```diff
@@ -58,6 +59,12 @@
     conda_include_dir = conda_prefix + '/include'
     conda_lib_dir = conda_prefix + '/lib'
 
+# Optional location of C++ build folder that can be configured by the user
```

Comment:
I would add some printing to inform developers how things are being compiled, you can use the `get_environment_option` of setuputils, that'll print a cmake style message of what options are being used at build time.

Reply:
The get_environment_option in the consequent line is indeed helpful for the user at build time. Also, added the update you mentioned for including the raft include directory.